src/components/Accordion/Accordion.stories.tsx

```diff
@@ -459,8 +459,8 @@ export const WithAdvancedFormControls: StoryFn<AccordionProps> = () => {
                     extra={
                         <Switch
                             label="advanced"
-                            on={showAdvancedSpacing}
-                            size={SwitchSize.Small}
+                            mode={showAdvancedSpacing ? 'on' : 'off'}
+                            size={'small'}
```

Comment:
```suggestion
                            size="small"
```
Same for the other occurences in the PR